integrations-and-supported-tools/fastai/scripts/Neptune_fastai_more_options.py

```diff
@@ -0,0 +1,44 @@
+import fastai
+from neptune_fastai.impl import NeptuneCallback
```

Comment:
it should be added (and imported) from neptune-client

from neptune.new.integrations.fastai import NeptuneCallback